happi/cli.py

```diff
@@ -392,6 +426,389 @@ def transfer(ctx, name: str, target: str):
     transfer_container(client, item, target)
 
 
+benchmark_sort_keys = [
+    'name',
+    'avg_time',
+    'iterations',
+    'tot_time',
+    'max_time',
+    'import_time',
+]
+
+
+@happi_cli.command()
+@click.pass_context
+@click.option("-d", "--duration", type=float, default=0,
+              help="Specify how long in seconds to spend per device.")
+@click.option("-i", "--iterations", type=int, default=0,
+              help="Specify the number of times to instantiate each device.")
+@click.option("-w", "--wait-connected", is_flag=True,
+              help="Wait for the devices to be connected.")
+@click.option("-t", "--tracebacks", is_flag=True,
+              help="Show tracebacks from failing device loads.")
+@click.option("-s", "--sort-key", type=str, default="avg_time",
+              help=(
+                "Sort the output table. Valid options are "
+                f"{', '.join(benchmark_sort_keys)}"
+              ))
+@click.option('--glob/--regex', 'use_glob', default=True,
+              help='Use glob style (default) or regex style search terms. '
+              r'Regex requires backslashes to be escaped (eg. at\\d.\\d)')
+@click.argument('search_criteria', nargs=-1)
+def benchmark(
+    ctx,
+    duration: float,
+    iterations: int,
+    wait_connected: bool,
+    tracebacks: bool,
+    sort_key: str,
+    use_glob: bool,
+    search_criteria: List[str],
+):
+    """
+    Compare happi device startup times.
+
+    This will generate a table that shows you how long each device took
+    to instantiate.
+
+    Repeats either for a max (DURATION) per device or for a fixed number or
```

Comment:
```suggestion
    Repeats either for a max (--duration) per device or for a fixed number or
```
Maybe?

Reply:
You're right, this is clearer. I was copying text from previous docstrings and never revisited.